config/routes.rb

```diff
@@ -53,7 +53,6 @@
     post 'data_entry', to: 'patients#data_entry_create', as: 'data_entry_create' # temporary
 
     resources :accountants, only: [:index, :edit]
-    post 'accountant/search', to: 'accountants#search', defaults: { format: :js }
```

Comment:
oh ok, so what was your rationale for removing the post?

Reply:
This was actually not something I intended to do at the start, but it felt like it made a lot of sense as I progressed! Essentially went as follows:
* kaminari async causes a LOT of weird problems
* we COULD, in theory, do this all with URL parameters and redirects?
* If that's the case, we don't need a search endpoint, and can just filter using URL params on the base endpoint, I think?
* omg it works

It has the side effect of making searches copy-pasteable to others, since you can set them via URL param rather than post payload.